src/lib.rs

```diff
@@ -1073,6 +1081,18 @@ where
         let node_iter = NodeIter::new(table, guard);
         Values { node_iter, guard }
     }
+
+    /// Exntends the [`FlurryHashMap`] with the contents of `iter`.
+    // TODO: Implement this in a not as stupid way
+    pub fn extend<'g, T: IntoIterator<Item = (K, V)>>(
```

Comment:
I think we want to also implement the [`Extend` trait](https://doc.rust-lang.org/std/iter/trait.Extend.html) for `&FlurryHashMap` now that we have this.

Reply:
The problem is that since `FlurryHashMap::put` requires a guard, the `FlurryHashMap::extend` does so too. Because of that we're not able to match `Extend::extend`'s type signature (I think).

Reply:
The reason why we're able to get arround this guard thing in the implementation of `FromIterator` is that the method `FromIterator::from_iter` is garanteed to be called only at the object's initialization, thus we can use `crossbeam::epoch::unprotected()` (sinse data races are not a problem)

Reply:
You can just create a `Guard` with `epoch::pin` inside `extend`, no?

Reply:
You're right, `epoch::pin` seems to do the trick.